src/include/network/lan/address.rb

```diff
@@ -1593,14 +1546,72 @@ def AddressDialog
 
   private
 
-    def get_selected_bridges
+    # Checks if the current LanItem is a bridge or an enslaved interface part
+    # of a bridge. In case that some configuration is not correct then adapt it
+    # if the user accepts.
+    def check_bridge_ports_config
+      return false unless LanItems.type == "br"
```

Comment:
As I try to drop dependency on these "global" status variables I prefer to have new functions stateless (e.g. checking device type on-the-fly as you never know who forgot to reset or set it somewhere to something reasonable). It should also ease "big bang" refactoring of ```LanItems``` one day.